tests/integration_tests/scheduler/test_lsf_driver.py

```diff
@@ -53,6 +53,7 @@ async def test_submit(tmp_path):
     assert (tmp_path / "test").read_text(encoding="utf-8") == "test\n"
 
 
+@pytest.mark.timeout(5)
```

Comment:
We don't need timeout for integration tests. Consider that `bsub` on the real cluster might take a few minutes to actually work.